lib/ip_wrangler/iptables.rb

```diff
@@ -0,0 +1,222 @@
+module IpWrangler
+  class Iptables
+    $iptables_bin_path = '/usr/bin/sudo /sbin/iptables'
+    $awk_bin_path = '/usr/bin/awk'
+    $tail_bin_path = '/usr/bin/tail'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.